src/ApiResponseHelpers.php

```diff
@@ -14,6 +14,9 @@
 
 trait ApiResponseHelpers
 {
+
+    private ?array $_api_helpers_defaultSuccessData = ['success' => true];
```

Comment:
That name probably won't be used by any project anywhere, but I'm glad to hear suggestions